### DISCUSSION TITLE
Multiple apps for Todo mode

### DISCUSSION THREAD

**Discussion by apiccirilli** [rxhanson/Rectangle#348] [Ideas]
First, the new Todo mode is great - thank you! Would it be possible to allow for 2+ apps in the reserved sidebar space? I don't need the full vertical space for my todo app, and always have something else (e.g., a pomodoro timer) alongside:

```
+---------------------------------+
|                        |        |
|                        |  todo  |
|                        |        |
|                        +--------+
|                        |        |
|                        |  timer |
|                        |        |
+---------------------------------+
```

Hoping someone else might find this useful as well.

Thanks again for the great app, and taking up the reins after Spectacle.

**Comment by shantanu-adjust** [comment]
@rxhanson If you could point me in the right direction, I could look into creating a PR!

**Reply by rxhanson** [reply]
Sure thing, start with the `Todo Manager` class. 

Here's the initial commit where I took all of Patrick & Nat's changes and rolled it in: https://github.com/rxhanson/Rectangle/commit/300995a1a128a23d778ca1bfd7fb834459c0ea2e
